ROADMAP.md

```diff
@@ -1,45 +1,42 @@
-    - **Add immutable caching to new generated code**. Caching is currently heavily tied into our existing parsing mechanism. We're going to separate that out in two phases: The first will allow caching that *cannot* be changed by the consumer.
-    - **Add mutable caching to new generated code**. This is the final stage of updating caching: Allowing caching that *can* be changed by the consumer.
-    - **Remove Old Codegen**. Once all this is built, older codegen will be deprecated. 
-
-    You can follow this effort through the [Swift Codegen GitHub project](https://github.com/apollographql/apollo-ios/projects/2).
-    
-- **Cache Improvements**: We've had a ton of requests for this and we've got some extra muscle now, so we're going to be working on many requests. First up, some features inspired by Apollo Client 3 for the web: 
-    - **Better pagination support**. Better support for caching and updating paginated lists of objects. 
-    - **Reducing over-normalization**. Only separating out results into individual records when something that can identify them is present
-    - **Codable Support**: Which will allow easier moving between any storage method and objects, along with working better with updated codegen.
-
-- **General bug-bashing**: There are always a few outstanding general issues and small feature requests I'd like to address, and we'll be dealing with any new issues as they come up.
+These are the major initiatives planned for 1.0.
+- **Swift Codegen Rewrite**: The code generation is being rewritten with a Swift-first approach instead of relying on scripting and Typescript. This will allow easier community contribution to code generation and provide the opportunity to improve various characteristics such as generated code size and performance.
+    - We are finishing the spec at the moment which will then be published as an RFC for review and feedback.
+    - Once we're ready to begin implementation those details and development phases will be shared here.
 
+- **Networking Stack Improvements**: The goal is to simplify and stabilise the networking stack.
+    - The [updated network stack](https://github.com/apollographql/apollo-ios/issues/1340) solved a number of long standing issues with the old barebones NetworkTransport but still has limitations. Switching to a composable request chain would enable building a pipeline out of reusable and customizable parts, giving users full control over the steps that are invoked to satisfy requests. This pattern has proven really useful for the JavaScript client (Apollo Link), and has been used succesfully in a number of other projects (like [OkHttp](https://square.github.io/okhttp/)).
 
-## Long-Term
+- **Improved Documentation and Tutorials**:
```

Comment:
Is DocC really a blocker for 1.0?

Reply:
No, I don't believe it is.

Reply:
DocC itself shouldn't be but better documentation is absolutely necessary